tests/e2e/e2e_common.go

```diff
@@ -44,6 +44,7 @@ const (
 	diskSizeInMinMb                            = int64(200)
 	e2eTestPassword                            = "E2E-test-password!23"
 	e2evSphereCSIDriverName                    = "csi.vsphere.vmware.com"
+	envCSINamespace                            = "CSI_NAMESPACE"
```

Comment:
@sashrith I have introduced a new env var for E2E tests called `CSI_NAMESPACE`. After this PR is merged, this will be a required variable to run vanilla tests. For dev testing and CSI 2.2 and above, this should be set to `vmware-system-csi`. For all versions below CSI 2.2, this should be set to `kube-system`